lib/bolt/error.rb

```diff
@@ -127,4 +127,10 @@ def initialize(msg, path)
       super(msg, 'bolt/file-error', { "path" => path })
     end
   end
+
+  class ConvertError < Bolt::Error
```

Comment:
In general we've been adding these into the classes that interact with them rather then further expanding this file.